exercises/transpose/transpose_test.go

```diff
@@ -0,0 +1,44 @@
+package transpose
+
+import (
+	"fmt"
+	"testing"
+)
+
+// For a more visual representation of the test data than is provided by
+// cases_test.go have a look at the original json data here:
+// https://github.com/exercism/x-common/blob/master/exercises/transpose/canonical-data.json
+
+func TestTranspose(t *testing.T) {
+	for _, test := range testCases {
+		actual := Transpose(test.input)
+		if fmt.Sprintf("%#v", actual) != fmt.Sprintf("%#v", test.expected) {
```

Comment:
yeah sure this seems like it has a better chance of being correct. There could be cases I'm not aware of, in which case we may get forced to use https://golang.org/pkg/reflect/#DeepEqual instead, but I at least can't think of a false positive on this one